gtk4/src/subclass/widget.rs

```diff
@@ -1298,6 +1298,12 @@ where
                 .expect("Failed to retrieve template child. Please check that it has been bound.")
         }
     }
+
+    /// Determines if the child has been bound. This is primarily
+    /// useful for implementing the Buildable interface.
+    pub fn is_bound(&self) -> bool {
```

Comment:
This should come together with a `fn try_get(&self) -> Option<T>` as otherwise you'll end up with code like

```rust
if self.foo.is_bound() {
  self.foo.stuff();
}
```

while

```rust
if let Some(foo) = self.foo.get() {
   foo.do_stuff();
}
```

would be nicer.

Reply:
Added
```rust
    /// Returns Some(child) if the widget has been bound.
    pub fn try_get(&self) -> Option<T> {
        unsafe { Option::<T>::from_glib_none(self.ptr) }
    }
```